internal/cli/alerts/settings_create.go

```diff
@@ -37,14 +37,16 @@ func (opts *ConfigCreateOpts) initStore() error {
 	return err
 }
 
+var createTemplate = "Creating alert configuration {{.ID}}.\n"
```

Comment:
[nit] for clusters we use "Creating" as it's async here you can say just "created"
```suggestion
var createTemplate = "Alert configuration '{{.ID}}' created with.\n"
```